test/functional/answers_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class AnswersControllerTest < ActionController::TestCase
```

Comment:
Actually the create and delete actions involve ajax requests. So I wanted to have exclusive tests for them using jasmine if possible. But writing tests with jasmine will take me some time.